### PR TITLE
[Routing][ServiceRouterLoader] Remove an outdated comment

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Since https://github.com/symfony/symfony/commit/0043653ea86c065547978a16619bf40932345c3e, this comment is technically false.

### CODE REVIEW

File: src/Symfony/Component/Routing/composer.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
         "symfony/yaml": "For using the YAML loader",
         "symfony/expression-language": "For using expression matching",
         "doctrine/annotations": "For using the annotation loader",
-        "symfony/dependency-injection": "For loading routes from a service"
+        "symfony/dependency-injection": "For using the routing resolver compiler pass"
```
Comment (fancyweb) [inline src/Symfony/Component/Routing/composer.json]: Not sure about this one : do we promote other components, or tell what can be done with them?

Comment (fabpot) [reply src/Symfony/Component/Routing/composer.json]: I would remove it altogether.

**Comment by fabpot** [comment]
Thank you @fancyweb.

File: src/Symfony/Component/Routing/composer.json (hunk 1 above)
Comment (fancyweb) [reply src/Symfony/Component/Routing/composer.json]: Didn't get the chance to do it 😅 @javiereguiluz This could be a good "first contribution issue".

Comment (fabpot) [reply src/Symfony/Component/Routing/composer.json]: I've done it :)

**Pull request merged**